### PR TITLE
Add happy path spec for password input widget

### PR DESCRIPTION
**Pull request by ajith-k-v**
Fixes: #4076 
Screenshot 2022-09-26 at 10 02 14 AM

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2uw67d1 Add HappyPath Spec for Password Input](https://app.clickup.com/t/2uw67d1)

File: cypress-tests/cypress/e2e/editor/widget/passwordInputHappyPath.cy.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    data.customText = randomString(12)
+
+    cy.renameApp(data.appName);
+
+    openEditorSidebar(passwordInputText.defaultWidgetName);
+    editAndVerifyWidgetName(data.widgetName);
+
+    openEditorSidebar(data.widgetName);
+    openAccordion(commonWidgetText.accordionProperties);
+    verifyAndModifyParameter('Placeholder', data.customText);
```
Comment (emidhun) [inline cypress-tests/cypress/e2e/editor/widget/passwordInputHappyPath.cy.js:54]: can we add validation for the default placeholder before modifying that?

**Review by emidhun — CHANGES_REQUESTED** [review]

**Review by emidhun — CHANGES_REQUESTED** [review]
I think we can remove the unused methods from the import section,
and the spec is failing.
Screenshot 2022-10-11 at 1 41 08 PM
Please fix this and let me know.

**Comment by ajith-k-v** [comment]
@emidhun Added review changes. Please review

Screenshot 2022-10-12 at 10 13 55 AM

**Review by emidhun — APPROVED** [review]
Looking Good To Me.
@ajith-k-v  Thank you for the changes.

**Pull request merged**